src/ngraph/runtime/cpu/builder/quantized_conv.cpp

```diff
@@ -73,20 +78,22 @@ namespace ngraph
                                     arg0_buffer_index,
                                     arg1_buffer_index,
                                     arg2_buffer_index,
+                                    arg4_buffer_index,
+                                    arg6_buffer_index,
                                     out0_buffer_index](CPURuntimeContext* ctx,
                                                        CPUExecutionContext* ectx) mutable {
                         // Create MKLDNN convolution primitive during the first iteration.
                         // Assumes the scales dont change for the duration of the graph
                         if (ctx->first_iteration)
                         {
                             vector<float> dyn_scales;
-                            dyn_scales.assign(
-                                static_cast<float*>(ctx->buffer_data[arg2_buffer_index]),
-                                static_cast<float*>(ctx->buffer_data[arg2_buffer_index]) +
-                                    scales_size);
+                            // Calculate the requantization scale
```

Comment:
requantization scale could be a vector of values. You seem to be only picking up the first value.

Reply:
From https://intelpedia.intel.com/NGraphDesign/Quantization 
---Add axes to allow for non-scalar scale and zero point (**OK to force to scalar for now)**
didn't we agree to assume scale to be scalar for now?

Reply:
it still needs to take in as vector, but you can force/assume to support only vector[0] ? (ok to assume/force to single/scalar val). Easier to extend if we make it vector.

Reply:
It is being stored in a vector and passed to mkldnn (because mkldnn expects a vector).